src/crepe/system/ParticleSystem.cpp

```diff
@@ -1,62 +1,143 @@
 #include <cmath>
+#include <cstdlib>
 #include <ctime>
 
-#include "../ComponentManager.h"
-#include "../api/ParticleEmitter.h"
+#include "api/ParticleEmitter.h"
+#include "api/Transform.h"
+#include "api/Vector2.h"
 
+#include "ComponentManager.h"
 #include "ParticleSystem.h"
 
 using namespace crepe;
 
-ParticleSystem::ParticleSystem() : elapsed_time(0.0f) {}
+ParticleSystem::ParticleSystem() {}
```

Comment:
Dit is een default implementatie van de constructor. Daarom moet dit in de header (zie code standaard). Maar is een default constructor uberhaupt wel nodig? Kan die niet gewoon weg?